mitmproxy/options.py

```diff
@@ -30,19 +30,19 @@ def __init__(
             app_port: int = APP_PORT,
             anticache: bool = False,
             anticomp: bool = False,
-            client_replay: Sequence[str] = (),
+            client_replay: Sequence[str] = [],
```

Comment:
That would impose a fiddly penalty throughout the codebase without much benefit. It would also not give us a clear consistent way forward if (when) we want to expand config to include maps.